pypff-stubs/pypff/attachment.pyi

```diff
@@ -0,0 +1,37 @@
+"""
+Type stubs for pypff_attachment defined in [1]
+
+[1] https://github.com/libyal/libpff/blob/main/pypff/pypff_attachment.c
```

Comment:
nit: maybe this should be changed to either the Everlaw repo (which would match with the version) or just to a file path relative to the repo's root?

Reply:
this should be changed to the everlaw repo

Reply:
[0b1e94c](https://github.com/Everlaw/libpff/pull/5/commits/0b1e94c3bc178a5877764ca98c6cc99fc145c698)